### DISCUSSION TITLE
MicroPython Embedding

### DISCUSSION THREAD

**Discussion by drorgl** [micropython/micropython#10008] [Core Development]
Hi,

I've recently did a POC about embedding MicroPython as a scripting engine and decided it was worth a summary of the experience.

I'd love your feedback. I apologize in advance if this is not the place to ask for feedback.

[https://blog.drorgluska.com/2022/11/embedding-micropython-in-your-projects.html](https://blog.drorgluska.com/2022/11/embedding-micropython-in-your-projects.html)

Thank you,
Dror

**Comment by jimmo** [comment]
Thanks @drorgl this is excellent!

FYI improving this is something that's on the radar: see https://github.com/micropython/micropython/pull/9529 which I think would have been quite useful for your project.
